deploy/rbac-clowdapp.yml

```
@@ -3447,6 +3479,18 @@ parameters:
 - name: PRINCIPAL_CLEANUP_DELETION_ENABLED
   description: Allow or disallow cleanup job to delete principals
   value: 'False'
+- name: PRINCIPAL_CLEANUP_DELETION_ENABLED_UMB
+  description: Allow cleanup job to delete principals via messages from UMB
+  value: 'False'
+- name: UMB_HOST
+  description: Host of the UMB service
+  value: 'localhost'
+- name: UMB_PORT
+  description: Port of the UMB service
+  value: 61612
```

the value should be string?
```suggestion
  value: '61612'
```

---

Nice catch, thanks!